.github/workflows/codeql-analysis-linux.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL for Linux"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '23 11 * * 6'
```

Comment:
Not sure why you need a cron job here (if this line does what I think it does.)

Reply:
I guess it's an extra "catch-all" - as this is a security check, it's presumed you do actually want it run and prove it's still going to work, at least once a week.